testing/camunda-process-test-java/pom.xml

```diff
@@ -57,6 +57,32 @@
       <artifactId>camunda-client-java</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <scope>compile</scope>
```

Comment:
I thought the same. :sweat_smile: If I remove scope, the maven-dependency-plugin complains that the dependency is defined for `test` scope. I guess there is a transient dependency from another dependency. Setting the scope to `compile` makes the plugin happy. :laughing:

Reply:
Ah, it's because the parent pom defines it with a test scope.

```xml
      <dependency>
        <groupId>org.awaitility</groupId>
        <artifactId>awaitility</artifactId>
        <version>${version.awaitility}</version>
        <scope>test</scope>
      </dependency>
```